lib/mayaUsd/utils/jsonDict.h

```diff
@@ -0,0 +1,216 @@
+//
+// Copyright 2024 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#ifndef MAYAUSD_UTILS_JSON_DICT_H
+#define MAYAUSD_UTILS_JSON_DICT_H
+
+#include <mayaUsd/base/api.h>
+
+#include <pxr/base/js/json.h>
+#include <pxr/base/vt/dictionary.h>
+#include <pxr/base/vt/value.h>
+
+#include <string>
+#include <vector>
+
+namespace MAYAUSD_NS_DEF {
+
+////////////////////////////////////////////////////////////////////////////
+//
+// Convert a VtDictionary to JsValue.
+// Modeled after PXR_NS::JsValueTypeConverter
+
+class VtDictionaryToJsValueConverter
```

Comment:
USD library provides a JSON -> VtDictionary converter, but not one going the other way. This is the other way. Not everything is supported, for example array-of-arrays is not supported and other such things.

Reply:
if this is more like a helper class may be we could rename the file to reflect the class itself.